### PR TITLE
Added SSD MobileNet V2 / SSD MobileNet V2 FPN Lite 320x320 Example And Close Issue #134

### PR DESCRIPTION
**Pull request by qu-ngx**
I happened to work with SSD MobileNet V2 and I found that a lot of people are getting trouble implementing this model in their application #134 . This pull request will introduce solution to #134 and an implementation example of SSD MobileNet V2 and SSD MobileNet V2 FPN Lite 320x320. Feel free to notify me if my code needs any further modifications.

### CODE REVIEW

**Comment by qu-ngx** [comment]
@PaulTR P @CaptainDario @st-duymai  cap can you take a look? Thank you!

**Comment by qu-ngx** [comment]
@PaulTR @st-duymai @CaptainDario can you guys take a look at my PR? Thank you so much!

**Comment by qu-ngx** [comment]
I talked to @PaulTR and he is currently on parental leave. Can @st-duymai check it for me? I want to open another pull request but this one is still not closed yet. Thanks a lot!

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (st-duymai) [inline .vscode/settings.json:1]: Should add this file into gitignore.

Comment (qu-ngx) [reply .vscode/settings.json:1]: I will do it now. Tks!

Comment (qu-ngx) [reply .vscode/settings.json:1]: I tried adding .vscode to gitignore but it doesn't seem to work on my machine so I removed them manually. Can you review my changes? Tks

Comment (st-duymai) [reply .vscode/settings.json:1]: The reason it won't work because you don't clear file cache yet. `git rm --cached filename`

Comment (qu-ngx) [reply .vscode/settings.json]: Oh tks! That one works. Sorry I'm inexperienced with this.

Comment (st-duymai) [reply .vscode/settings.json]: Nothing. There are a lot of files. It will take me some time. When I finish reviewing it I will let you know. Thank.

Comment (qu-ngx) [reply .vscode/settings.json]: Tks. Take your time. Remind me if something does not work properly. I will try to resolve asap

**Pull request merged**